tests/page_objects/care_platform_page.yaml

```diff
@@ -209,6 +209,8 @@ care_platform_edit_participant_records:
   changes_saved_notification: //div[@class='Toastify']//div[contains(text(),'Participant changes saved')]
   medication_saved_notification: //div[@class='Toastify']//div[contains(text(),'Medication created')]
   medication_deleted_notification: //div[@class='Toastify']//div[contains(text(),'Medication deleted')]
+  nav_vendor_button: //ul[@class='form-navigation']//li/button[text()='Vendors']
```

Comment:
please change the var name to use the format of the other ones, it could be "vendor_navigation".

Reply:
it' s solved in this commit https://github.com/neveralone-chs-org/TestRay/pull/161/commits/c7caf4eb16523176638f191bde17940b5a29af59